src/Models/Services/TelemetryService/Telemetry.ts

```diff
@@ -2,12 +2,20 @@ import {
     CustomProperties,
     IBaseTelemetryParams,
     IExceptionTelemetryParams,
+    IMetricTelemetryParams,
     IRequestTelemetryParams,
     ITraceTelemetryParams,
     SeverityLevel,
     TelemetryType
 } from './TelemetryService.types';
 
+export type ITelemetryEvent =
```

Comment:
Yeah, can we rename this? Since there is type EventTelemetry, seeing ITelemetryEvent might be confusing, I know it is the "event" object but maybe we can rename it to `ITelemetryObject` or something?

Reply:
Ok. renamed to `ITelemetryItem`. I'd rather avoid ITelemetry as that doesn't really tell you whether it's the service, a single item, arguments, or whatnot. Let me know if you think `ITelemetryObject` is better.